flask_sqlalchemy/__init__.py

```diff
@@ -577,7 +577,7 @@ def _should_set_tablename(bases, d):
             return False
 
         for name in dir(base):
-            attr = getattr(base, name)
+            attr = getattr(base, name, None)
```

Comment:
where is the unittest that demonstrates that this helps

as far as i understood getattr will still fail in basically all computing exceptions (and rightfully so)